docker/project.go

```diff
@@ -68,3 +77,40 @@ func NewProject(context *Context, parseOptions *config.ParseOptions) (project.AP
 
 	return p, err
 }
+
+// DockerProject implements project.RuntimeProject and define docker runtime specific methods.
```

Comment:
This might be better off in a package by itself (like `docker/project` or something) or at least a new file :angel:.

This also need unit tests :stuck_out_tongue_closed_eyes: